packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,439 @@
+
+You may also notice that we included an info button in the rows in our example. This is to highlight `useList`'s ability to handle interactive elements inside a row.
+This differs from [useListBox](useListBox.html), which is a selectable list with no support for interactive elements. If your list will not contain interactive elements,
+[useListBox](useListBox.html) may be a better choice due to better screen reader experience, especially on mobile devices.
+
+```tsx example export=true
+import {useListState} from '@react-stately/list';
+import {mergeProps} from '@react-aria/utils';
+import {useRef} from 'react';
+import {useFocusRing} from '@react-aria/focus';
+import {useButton} from '@react-aria/button';
+import {useList, useListItem} from '@react-aria/list';
+import { Item } from "@react-stately/collections";
+
+function List(props) {
+  let state = useListState(props);
+  let ref = useRef();
+  let { collection } = state;
+  let { gridProps } = useList(props, state, ref);
+
+  return (
+    <ul
+      {...gridProps}
+      ref={ref}
+      style={{
+        padding: 0,
+        margin: '5px 0',
+        listStyle: 'none',
+        border: '1px solid gray',
+        maxWidth: 400
+      }}
+    >
+      {[...collection].map((item) => (
+        <ListItem key={item.key} item={item} state={state} />
+      ))}
+    </ul>
+  );
+}
+
+function ListItem({ item, state }) {
+  let ref = React.useRef();
+
+  let {
+    rowProps,
+    gridCellProps,
+    isPressed,
+    isSelected,
+    isFocused,
+    isDisabled
+  } = useListItem(
+    {
+      node: item
+    },
+    state,
+    ref
+  );
+  let {isFocusVisible, focusProps} = useFocusRing();
+  let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
+
+  let backgroundColor;
+  let color = "black";
+
+  if (isSelected) {
+    backgroundColor = "blueviolet";
+    color = "white";
+  } else if (isFocused) {
+    backgroundColor = "gray";
+  } else if (isDisabled) {
+    backgroundColor = "transparent";
+    color = "gray";
+  }
+
+  return (
+    <li
+      {...mergeProps(rowProps, focusProps)}
+      ref={ref}
+      style={{
+        background: isSelected
+          ? 'blueviolet'
+          : isPressed
+            ? 'var(--spectrum-global-color-gray-400)'
+            : item.index % 2
+              ? 'var(--spectrum-alias-highlight-hover)'
+              : 'none',
+        color: isSelected ? 'white' : null,
+        padding: '2px 5px',
+        outline: isFocusVisible ? '2px solid orange' : 'none'
+      }}
+    >
+      <div {...gridCellProps}>
+        {showCheckbox && <ListCheckbox item={item} state={state} />}
+        {item.rendered}
+        <span style={{float: 'right'}}>
+          <Button onPress={() => alert(`Info for item ${item.key}`)}>Info</Button>
```

Comment:
I think this is ok since we call it out above, but I wonder whether it's also clear that user content (i.e. children of `<Item>`) may also contain interactive children? I guess that might be more common than a hard coded interactive element within every item?

Reply:
Updated to use Button as a child in a few items on the page. I think this is a lot cleaner than every item, and is probably more common as you mentioned.